src/tools/compiletest/src/runtest/tests.rs

```diff
@@ -48,3 +48,73 @@ fn normalize_platform_differences() {
         r#"println!("test\ntest")"#,
     );
 }
+
+/// Test for anonymizing line numbers in coverage reports, especially for
+/// branch regions. This test can be removed when we have examples of branch
```

Comment:
nit: It might be nice to have a `FIXME: ` note for this which will make it a bit easier to search for